### ISSUE TITLE
how can i train it?

### ISSUE THREAD

**Issue by liangzhenlin123** [carlthome/tensorflow-convlstm-cell#20]
I would like to use convlstm in speech, but the training effect is very poor (I use adam), and the recognition rate in the test set fixed at 0.5844444 remain unchanged, I made a lot of ways or no way to change, can you send a Damo train this network, such as handwritten numbers identify.
Thank u very much!

**Comment by carlthome** [comment]
Do you mean that the test accuracy is precisely the same regardless of your model? That indicates a bug in your code. The first thing you want to do is make sure you can overfit and get close to 100% training accuracy. Can you?
